cmd/dp-recipe-api/main.go

```diff
@@ -144,7 +145,7 @@ func main() {
 			hasShutdownError = true
 		}
 
-		if enableMongoData {
+		if enableMongoData || enableMongoImport || enableAuthImport {
```

Comment:
As there are a number of flags here, consider creating a single variable for the value and reusing that. Something like this:
```
mongoIsEnabled := enableMongoData || enableMongoImport || enableAuthImport
```

Reply:
Done in commit _Initialising all feature flags into single variable in main.go_